### ISSUE TITLE
Benim analysis doesn't return Ben+Pron+Gen...

### ISSUE THREAD

**Issue by sonerx** [ahmetaa/zemberek-nlp#115]
Benim doesn't return the genitive of the personal pronoun analysis. This is the most common analysis for this word and currently does not appear in the analyses.

**Comment by ahmetaa** [comment]
Yes. Zemberek's weakest point is Pronouns as their morphotactics are tricky and not fitting well to the current structure. Hopefully it will be fixed until 0.12.0

**Comment by sonerx** [comment]
Thanks Ahmet,

Is there any list for Pronouns which cannot be processed by Zemberek. It'll be great to know which tokens are problematic till you will find time and solve the cases.
